notifications/templates.ts

```diff
@@ -0,0 +1,45 @@
+import { MessageType } from './messageTypes';
+import { GraphQLUser } from '../common/user/session';
+import { Concrete_notification as ConcreteNotification } from '../graphql/generated';
+
+// only actionPath or url is optionally allowed but not both
+type NotificationAction = { navigateTo?: string; url?: undefined } | { url: string; navigateTo?: undefined };
+export type MessageTemplate = {
+    header: string;
+    body: string;
+    messageType: MessageType;
+} & NotificationAction;
+
+export const getMessage = (concreteNotification: ConcreteNotification, user: GraphQLUser): MessageTemplate => {
+    const { firstname, lastname } = user;
+    const { notificationID, context } = concreteNotification;
+
+    const templates: { [key: number]: MessageTemplate } = {
+        1: {
+            header: `bla bla ${firstname}`,
```

Comment:
So as a next step you would then move the MessageTemplates to the DB correct? Or how are you going to proceed here? (Good intermediate step though)

Reply:
we wanted to have them in a file, so they will stay here until we decide for a different solution (if we decide to change it)